### DISCUSSION TITLE
Adding More Apps

### DISCUSSION THREAD

**Discussion by MrKomail** [cea-sec/ivre#1619] [Q&A] [answered]
Hello everyone,

I've been tinkering with this network recon tool that uses nmap and masscan for the active recon part, and it's been pretty sweet. Now, I'm thinking of kicking it up a notch by adding Nikto into the mix.

Any thoughts on whether this could work? And if so, any tips on how to go about it? 

Thanks in advance for any insights!

**Comment by p-l-** [comment, marked as answer]
It could probably work yes. A good starting point could be to have a look at what currently exists for Nuclei. Let me know if you need help (I'd suggest that you start with a simple `git grep -i nuclei` and a bit of code reading).
